components/SpotlightModal/styles.css

```diff
@@ -0,0 +1,37 @@
+/* index.css */
+.spotlight-modal {
+  font-size: 12px;
+}
+.spotlight-modal > .content {
+  width: 100%;
+  padding: 10px 5px;
+}
+.spotlight-modal > .actions {
+  margin: auto;
+}
+.spotlight-modal > .actions {
+  width: 100%;
+  padding: 10px 5px;
+  text-align: center;
+}
+.content > .description {
```

Comment:
Replace all this with css-in-js ... i.e. <style jsx>` ... easier to track and great with server-side rendering... it's also consistent. I don't see any class called `actions` being used anywhere.

Reply:
Updated to use css-in-js